tools/mmp/driver.cs

```diff
@@ -274,6 +274,7 @@ static void Main2 (string [] args)
 						}
 					}
 				},
+				{ "linkplatform", "Link only the Xamarin.Mac.dll platform assembly", v => App.LinkMode = LinkMode.Platform },
```

Comment:
What about XamMac.dll? Do we want to support Classic? We might want to disallow it, just to minimize the test matrix.